app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java

```diff
@@ -241,17 +241,6 @@ public void onConfigurationChanged(Configuration newConfig) {
     }
 
     @Override
-    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        // If the drawer is open, show the global app actions in the action bar. See also
-        // showGlobalContextActionBar, which controls the top-left area of the action bar.
-        if (mDrawerLayout != null && isDrawerOpen()) {
-            inflater.inflate(R.menu.global, menu);
-            showGlobalContextActionBar();
-        }
```

Comment:
I added it back in a later commit. It's in the refactoring PR I'm pretty sure. I just wanted to remove the menu when the nav drawer popped up. Should've just deleted L248